Raycasting/src/Shader.cpp

```diff
@@ -3,46 +3,22 @@
 #include <iostream>
 #include <sstream>
 
-Shader::Shader(std::ifstream & fileStream) {
```

Comment:
this constructor was replaced with one that uses string input so it can dynamically initialize later on. Have no fear! We can still read shaders from files!